extensions/wikia/UserActivity/UserActivityController.class.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace UserActivity;
+
+class Controller extends \WikiaController {
+	const DEFAULT_TEMPLATE_ENGINE = \WikiaResponse::TEMPLATE_ENGINE_MUSTACHE;
+	const NUM_ARTICLES_SHOWN = 3;
+
+	public function index() {
+		$user = \F::app()->wg->User;
+		$limit = $this->getVal( 'limit' );
+		$offset = $this->getVal( 'offset' );
+		$order = $this->getVal( 'order' );
+
+		$result = [
+			'total' => 0,
+			'totalReturned' => 0,
+			'items' => []
+		];
+
+		if ( empty( $user ) ) {
+			return;
+		}
+
+		if ( $user->isBlocked() ) {
+			return;
+		}
+
+		if ( !$user->isLoggedIn() ) {
+			return;
+		}
+
+		$contribs = new \LookupContribsCore( $user->getName() );
+		if ( !$contribs->checkUser() ) {
+			$this->response->setData( $result );
+			return;
+		}
+
+		$contribs->setLimit( $limit );
+		$contribs->setOffset( $offset );
+
+		$activity = $contribs->checkUserActivity(
+			$addEditCount = true,
```

Comment:
Its to give context to what is otherwise a meaningless floating `true` and shorthand for setting this variable else where to pass here.